src/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -4022,7 +4108,7 @@ public void Shutdown(SocketShutdown how)
             if (NetEventSource.IsEnabled) NetEventSource.Exit(this);
         }
 
-        #region Async methods
+#region Async methods
```

Comment:
Actually VS seems to reformat in different ways for reasons that are not clear to me.

I've reverted all of these to the state they were in before.